app/src/main/res/values/styles.xml

```diff
@@ -480,6 +480,29 @@
         <item name="android:layout_height">28dp</item>
     </style>
 
+    <style name="ImageView.RoundedCorner" parent="">
+        <item name="cornerFamily">rounded</item>
+    </style>
+
+    <style name="ImageView.RoundedCorner.TopLeft">
```

Comment:
Notice that I'm adding styles for rounding each individual corner of a view. This will be necessary for design updates introduced in Rabbit Holes.